packages/contracts/src/interest/IMultiTokenVault.sol

```diff
@@ -226,25 +229,15 @@ interface IMultiTokenVault is IERC1155 {
      *
      * @return currentTimePeriodsElapsed The current number of time periods elapsed.
      */
-    function currentTimePeriodsElapsed() external view returns (uint256 currentTimePeriodsElapsed);
+    function currentTimePeriodsElapsed() external view returns (uint256);
 
     /**
      * @notice This function is for only testing purposes.
      * @dev This function is made to set the current number of time periods elapsed.
      *
-     * @param currentTimePeriodsElapsed The current number of time periods elapsed.
-     */
-    function setCurrentTimePeriodsElapsed(uint256 currentTimePeriodsElapsed) external;
-
-    /**
-     * @dev Returns the shares held by account for a specific deposit period.
-     *
-     * @param account The address that owns the minted shares.
-     * @param depositPeriod The current number of time periods elapsed.
-     *
-     * @return shares The amount of shares held by the account for the specific deposit period.
+     * @param currentTimePeriodsElapsed_ The current number of time periods elapsed.
      */
-    function sharesAtPeriod(address account, uint256 depositPeriod) external view returns (uint256 shares);
```

Comment:
Regarding this function, can we add it to the abstract class that inherits from this interface?